uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1016.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1016 {
+  /**
```

Comment:
We don't have have this in the coding standards, but most classes have an empty line after the opening brace, can you add one here?